syntax="proto2";

option optimize_for=SPEED;

package aap_protobuf.service.phonestatus;

enum PhoneStatusMessageId {
  PHONE_STATUS = 32769;
  PHONE_STATUS_INPUT = 32770;
}